tests/integration/targets/ec2_asg_instance_refresh/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+unsupported
```

Comment:
Is there a reason you're marking this tests unsupported?  This means that the new tests won't be automatically run in CI.

If there are missing permissions, then a pull request can be opened against the aws-terminator repo: https://github.com/mattclay/aws-terminator/pulls

Reply:
removed